### PR TITLE
Refs #41 Add some test for creme.commercial.setScore()

### PR DESCRIPTION
**Pull request by joehybird**

### CODE REVIEW

File: creme/commercial/static/commercial/js/commercial.js [hunk 1]
```diff
@@ -21,14 +21,15 @@
```
Comment (genglert) [inline creme/commercial/static/commercial/js/commercial.js:21]: Update the date in the header

File: creme/settings.py [hunk 2]
```diff
@@ -888,6 +888,7 @@
     ('creme.billing',       'billing/js/tests/billing.js'),
     ('creme.billing',       'billing/js/tests/billing-actions.js'),
     ('creme.billing',       'billing/js/tests/billing-listview.js'),
+    ('creme.commercial',    'commercial/js/tests/commercial-assets.js'),
```
Comment (genglert) [inline creme/settings.py:891]: "asset" is not very relevant ; there is asset-score, there is charm-score. It's all about "score", not "asset".

File: creme/commercial/static/commercial/js/commercial.js (hunk 1 above)
Comment (joehybird) [reply creme/commercial/static/commercial/js/commercial.js:21]: Good catch. The git hook only checks python files :sweat_smile:

File: creme/settings.py (hunk 2 above)
Comment (joehybird) [reply creme/settings.py:891]: You are right, `commercial-score.js` is a better name

**Pull request merged**